### ISSUE TITLE
Empty comment count links are selectable on Carousel cards

### ISSUE THREAD

**Issue by bryophyta** [guardian/dotcom-rendering#6111]
The issue here is the same as in #5634: Empty `<a>` tags are being rendered on cards which don't have any comments, because they are being created server-side and not being hydrated because e.g. commenting is not available for the article in question.

This was addressed for Fronts cards in #5635, by only rendering the empty hydration target if `isCommentable` was true on the `Trail` object. This didn't address the cards in Onwards carousels, however, because they use a different `TrailType`, which doesn't have `isCommentable`.

One solution is to add `isCommentable` to Onwards trails in Frontend. If doing this, it might make sense to delay work on this until Onwards has been refactored.

Another solution would be to add a non-selectable placeholder for comment counts, and insert the whole  `<a>` tag on hydration. I don't know what the overhead of this would be in terms of client-side js, and I also don't know if there is server-time data (e.g. about formats) which would need to be made available to the client-side hydration code.

**Comment by bryophyta** [comment]
hey @AshCorr -- I noticed that you had a PR for this but then closed it, do you remember what the status of work on this was?

**Comment by AshCorr** [comment]
Sorry @bryophyta!

Yea, that PR was an idea, it removed the count links in DCR by not rendering the links as links at all but as another placeholder element, but i think as you say we should try to make this change in frontend so that we're not telling DCR that there should be discussion links on things that dont have any

**Comment by VDuczekW** [comment]
we should like to do this before fronts launch due to affecting accessibility scores
